doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+          "accepted-status" : [ "CREATING", "STARTED", "FAILED", "DISPOSING", "STOPPED" ]
+        }
+      }
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "List of created containers",
+      "result": {
+        "kie-containers": {
+          "kie-container": [
+            {
+              "container-id": "command-script-container",
+              "release-id": {
+                "group-id": "com.redhat",
+                "artifact-id": "Project1",
+                "version": "1.0"
+              },
+              "resolved-release-id": {
+                "group-id": "com.redhat",
+                "artifact-id": "Project1",
+                "version": "1.0"
+              },
+              "status": "STARTED",
+              "scanner": {
+                "status": "STARTED",
+                "poll-interval": 5000
+              },
+              "config-items": [
+                {
+                  "itemName": "RuntimeStrategy",
+                  "itemValue": "SINGLETON",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "MergeMode",
+                  "itemValue": "MERGE_COLLECTIONS",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "KBase",
+                  "itemValue": "",
+                  "itemType": "java.lang.String"
+                },
+                {
+                  "itemName": "KSession",
+                  "itemValue": "",
+                  "itemType": "java.lang.String"
+                }
+              ],
+              "messages": [
+                {
+                  "severity": "INFO",
+                  "timestamp": {
+                    "java.util.Date": 1538504619749
+                  },
+                  "content": [
+                    "Container command-script-container successfully created with module com.redhat:Project1:1.0."
+                  ]
+                }
+              ],
+              "container-alias": null
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+--
+CallContainerCommand::
++
+--
+Calls a KIE container and executes one or more runtime commands. For information about {PRODUCT} runtime commands, see xref:runtime-commands-con_kie-apis[].
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container to be called
+|Required
+
+|`payload`
+|One or more commands in a `BatchExecutionCommand` wrapper to be executed on the KIE container
+|Required
+
+|
+|
+|Optional
+|===
+
+*<@Maciej/@Tiho: The above attributes table is missing the rest of the required and optional parameters, and the below example Java command contains only the required payload portion and needs the rest of the call-container command to be complete.>*
```

Comment:
is this what you had in mind or there are other things to be added?

Reply:
I guess so. I figured you'd tell me what was missing :) Done.